vms/registration/views.py

```diff
@@ -16,7 +16,7 @@
 from administrator.models import *
 
 
-class AdministratorSignupView(TemplateView):
+class AdministratorSignupView(AnonymousRequiredMixin, TemplateView):
```

Comment:
@smarshy Well... It is the default Mixin used to prevent logged in users to access that view; i thought the task said to make that view inaccessible to logged in users, so did that.